<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reader</class>
 <widget class="QWidget" name="reader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>读者页面</string>
  </property>
  <widget class="QLineEdit" name="leSearch">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSearch">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>70</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>已借书目：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>450</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>留言给我们：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="teMessage">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>470</y>
     <width>471</width>
     <height>78</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSubmit">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>560</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="twLend">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>571</width>
     <height>171</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>书号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>书名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>借阅日期</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>应还日期</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>606</x>
     <y>0</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRet">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>360</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>归还</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chbRet1">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>260</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>选中</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chbRet2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>290</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>选中</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chbRet3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>320</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>选中</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbBook">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbWriter">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbState">
   <property name="geometry">
    <rect>
     <x>306</x>
     <y>140</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbLend">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>140</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>借阅</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMod">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>0</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>修改个人资料</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>书名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>用户：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbUser">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
